ci/run_tests.sh

```diff
@@ -3,7 +3,7 @@
 # - PYTEST_BACKENDS: Space-separated list of backends to run
 # - PYTEST_EXPRESSION: Marker expression, for example "not udf"
 
-TESTS_DIRS="ibis/tests"
+TESTS_DIRS="ibis/tests ibis/backends/tests ibis/backends/base*/tests"
```

Comment:
I don't think this is actually picking up the tests in `ibis/backends/base*/tests`. If I look at tests running in https://github.com/ibis-project/ibis/pull/2576/checks?check_run_id=1651224874, I don't see tests from `ibis/backends/base_file/tests/test_schema.py` running (that seems to be the only `base*` that has `tests`. 

Were the files in https://github.com/datapythonista/ibis/tree/refactor_backend_tests/ibis/tests/sql also supposed to move to these `base_` dirs?

Reply:
I do see those tests running: https://github.com/ibis-project/ibis/pull/2576/checks?check_run_id=1651224874#step:4:98

> Were the files in https://github.com/datapythonista/ibis/tree/refactor_backend_tests/ibis/tests/sql also supposed to move to these base_ dirs?

Yes, I think so. There should be an issue for it if I'm not wrong.

Reply:
Oh - weird, not sure where I was looking that I missed that!